tests/basic_test.go

```diff
@@ -207,6 +202,7 @@ func TestSparkHistoryServerInstallation(t *testing.T) {
 		log.Infof("Spark History Server logs:")
 		utils.Kubectl("logs", "-n", spark.Namespace, historyServerPodName)
 	}
+
 	utils.AwsS3DeleteFolder(awsBucketName, awsFolderPath)
```

Comment:
will this function be called disregarding the previous errors? can we use it in `defer` after the folder is created?

Reply:
it would make sense to keep this files in case of errors for troubleshooting, but we already collecting logs if something goes wrong, so yes, let's change that